Source/mainmatter/negmodal.tex

```diff
@@ -0,0 +1,216 @@
+% The "proof.sty" and "bussproofs.sty" packages are available.
+% If you need any other package, please contact the coordinator (Bruno Woltzenlogel Paleo <bruno.wp@gmail.com>)
+
+\renewcommand{\arraystretch}{2}
+\centering\footnotesize
+\begin{tabular}{ccc@{\hspace{2em}}cc}
+
+~
+${[{\ineg}{\Ra}]}$
+& $\ssrul{\g\Ra\varphi, \d}{\uneg\d,\ineg\varphi\Ra \ineg\g} $
+&&
+${[{\Ra}{\uneg}]} $
+& $\ssrul{\g,\varphi\Ra \d}{\uneg\d\Ra \uneg\varphi,\ineg\g}$
+\\
+${[{{\wsmile}}{\Ra}]}$
+& $\ssrul{\g\Ra\varphi, \d\ \ \ \g\Ra\ineg\varphi,\d}
+{\g,{\wsmile}\varphi\Ra\d}$
+&&
+${[{\Ra}{{\wsmile}}]}$
+& $\ssrul{\g,\varphi,\ineg\varphi\Ra \d}{\g\Ra{\wsmile}\varphi,\d}$
+\\%[1mm]
+${[{{\wfrown}}{\Ra}]}$
+& $\ssrul{\g\Ra\varphi,\uneg\varphi,\d}
+{\g,\wfrown\varphi\Ra\d}$
+&&
+ ${[{\Ra}{\wfrown}]}$
+& $\ssrul{\g,\varphi\Ra \d\ \ \ \g,\uneg\varphi\Ra\d}{\g\Ra\wfrown\varphi,\d}$
+\\
+$[\Di]$
+& $\ssrul{\Gamma\Ra\Delta}{\uneg\Delta\Ra\ineg\Gamma}$
+ &&
+ $[\FUNC]$
+ &$\ssrul{\Gamma\Ra\Delta}{\ineg\Delta\Ra\ineg\Gamma}$
+\\
+$[\T_{1}]$
+& $\ssrul{\Gamma,\varphi\Ra\Delta}{\Gamma\Ra\ineg\varphi,\Delta}$
+&&
+ $[\T_{2}]$
+ & $\ssrul{\Gamma\Ra\varphi,\Delta}{\Gamma,\uneg\varphi\Ra\Delta}$
+\\
+$[\B_{1}]$
+& $\ssrul{\Gamma,\ineg\Gamma',\varphi\Ra\Delta,\uneg\Delta'}{\uneg\Delta,\Delta'\Ra\uneg\varphi,\ineg\Gamma,\Gamma'}$
+&&
+ $[\B_{2}]$
+ & $\ssrul{\Gamma,\ineg\Gamma'\Ra\varphi,\Delta,\uneg\Delta'}
+ {\uneg\Delta,\Delta',\ineg\varphi\Ra\ineg\Gamma,\Gamma'}$
+ \\
+$[\KFour_{1}]$
+& $\ssrul{\uneg\Gamma,\Gamma',\varphi\Ra\ineg\Delta,\Delta'}{\uneg\Gamma,\uneg\Delta'\Ra\uneg\varphi,\ineg\Delta,\ineg\Gamma'}$
+&&
+$[\KFour_{2}]$
+& $\ssrul{\uneg\Gamma,\Gamma'\Ra\varphi,\ineg\Delta,\Delta'}{\uneg\Gamma,\uneg\Delta',\ineg\varphi\Ra\ineg\Delta,\ineg\Gamma'}$
+\\
+$[\Di_{B}]$
+& $\ssrul{\ineg\Gamma',\Gamma\Ra\Delta,\uneg\Delta'}{\Delta',\uneg\Delta\Ra\ineg\Gamma,\Gamma'}$
+&&
+$[\Di_{4}]$
+& $\ssrul{\uneg\Gamma',\Gamma\Ra\Delta,\ineg\Delta'}{\uneg\Gamma',\uneg\Delta\Ra\ineg\Gamma,\ineg\Delta'}$
+\\
+\end{tabular}
+\normalsize
+
+
+
+\end{calculus}
+
+% The following sections ("clarifications", "history",
+% "technicalities") are optional. If you use them,
+% be very concise and objective. Nevertheless, do write full sentences.
+% Try to have at most one paragraph per section, because line breaks
+% do not look nice in a short entry.
+
+ \begin{clarifications}
+ $\mostbasic$ is obtained from propositional positive $\LK$,
+ by the addition of the rules for $\wsmile$ and $\wfrown$.
+ {$\GPK=\mostbasic+[\uneg\Ra]+[\ineg\Ra]$,}
+ {$\GPKD=\GPK+[\Di]$,}
+ {$\GPKT=\GPK+[\T_{1}]+[\T_{2}]$,}
+ {$\GPKF=\mostbasic+[\FUNC]+[\ineg=\uneg]$,}
+ {$\GPKB=\mostbasic+[\B_{1}]+[\B_{2}]$,}
+ {$\GPKFour=\mostbasic+[\KFour_{1}]+[\KFour_{2}]$,}
+ {$\GPKDB=\GPKB+[\Di_{\B}]$,} and
+ {$\GPKFourD=\GPKFour+[\Di_{\KFour}]$.}
+
+ \end{clarifications}
+
+ \begin{history}
+$\GPK$ was introduced in \cite{dod:mar:ENTCS2013}. All other systems were introduced in \cite{lahav_it_2016}.
```

Comment:
Thanks for the contribution, Yoni!

Could you add one or two short sentences here or in the clarification section, explaining and motivating negative modalities? (I know that sequent calculus experts will be able to figure this just by looking at the inference rules, but the encyclopedia's audience is a bit wider.)